source/opt/loop_descriptor.cpp

```diff
@@ -321,6 +327,26 @@ void Loop::SetPreHeaderBlock(BasicBlock* preheader) {
   loop_preheader_ = preheader;
 }
 
+ir::BasicBlock* Loop::FindLatchBlock() {
+  ir::CFG* cfg = context_->cfg();
+
+  opt::DominatorAnalysis* dominator_analysis =
+      context_->GetDominatorAnalysis(loop_header_->GetParent());
+
+  // Find the backedge to the header which should be dominated by the loop
```

Comment:
This phrasing is a bit confusing.  The backedge is not dominated by the continue target.  It's the latch block.  And the latch block contains the backedge to the header, right?

Reply:
Yeah it's a bit misleading, updated it now